### PR TITLE
Absorb token and helloworld submodules

### PR DESCRIPTION
**Pull request by yorhodes**
It is very painful to update any dependencies across repos and publish new releases. Fresh clones are also broken.

Removes token and helloworld submodules and merges in subrepos.

Followed https://nuclearsquid.com/writings/subtree-merging-and-you/ for preserving history.

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -29,9 +29,6 @@ jobs:
         with:
           submodules: recursive
 
-      - name: remove submodule locks
```
Comment (asaj) [inline .github/workflows/e2e.yml:32]: Is checkout with submodules, above, still relevant? Can't recall if we have foundry now as a submodule

File: .github/workflows/node.yml [hunk 2]
```diff
@@ -18,9 +18,6 @@ jobs:
         with:
           submodules: recursive
 
-      - name: remove submodule locks
```
Comment (asaj) [inline .github/workflows/node.yml:21]: Ditto

File: rust/utils/run-locally/src/main.rs [hunk 3]
```diff
@@ -297,7 +297,7 @@ fn main() -> ExitCode {
         // don't need to clean in the CI
         build_cmd(&["yarn", "clean"], Some("../"), None);
     }
-    build_cmd(&["yarn", "build:e2e"], Some("../"), None);
+    build_cmd(&["yarn", "build"], Some("../"), None);
```
Comment (asaj) [inline rust/utils/run-locally/src/main.rs:300]: Why not build:e2e anymore?

**Review by asaj — APPROVED** [review]
Thanks for doing this

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (yorhodes) [reply .github/workflows/e2e.yml:32]: yes, still needed for foundry

File: rust/utils/run-locally/src/main.rs (hunk 3 above)
Comment (yorhodes) [reply rust/utils/run-locally/src/main.rs:300]: we don't need separate builds

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (yorhodes) [reply .github/workflows/e2e.yml:32]: you can see it in `.gitmodules`

**Pull request merged**